stacks/odincal_stack.py

```diff
@@ -781,101 +780,53 @@ def set_up_create_zpt(
 
     def set_up_activate_level2(
         self,
-        lambda_timeout: Duration = Duration.seconds(900),
     ) -> sfn.State:
-
-        # Set up Lambda functions
-        activate_level2_lambda = Function(
+        map_state = sfn.Map(
             self,
-            "OdinSMROdincalActivateLevel2Lambda",
-            code=InlineCode.from_asset("activate_l2"),
-            handler="handler.activate_l2_handler.activate_l2_handler",
-            timeout=lambda_timeout,
-            architecture=Architecture.X86_64,
-            runtime=Runtime.PYTHON_3_10,
+            "OdincalArrangeL2",
```

Comment:
```suggestion
            "OdinSMROdincalArrangeL2",
```
Alla andra heter så.